lib/sentry/client.ex

```diff
@@ -157,6 +163,22 @@ defmodule Sentry.Client do
     {endpoint, public_key, secret_key}
   end
 
+  def maybe_call_after_send_event({:ok, _} = result, event) do
```

Comment:
@jeregrine Right now we only try to call the hook if the sending event was successful, but I was thinking since we're passing the result, we could call `function.(result, event)` and let the user decide so we can skip the logic on our side?